admin/migrate-to-deb.sh

```diff
@@ -0,0 +1,124 @@
+fi
+echo "- Old config directory found: $OLD_CONFIG_DIR"
+
+# Verify root script exist
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_SCRIPT="$SCRIPT_DIR/configure-tails-persistence.sh"
+if [[ ! -f "$ROOT_SCRIPT" ]]; then
+    error_exit "Helper script not found.\n\nExpected: $ROOT_SCRIPT"
+fi
+
+# Run the root script
+echo "Configuring Tails persistence (requires password)..."
+if ! pkexec bash "$ROOT_SCRIPT"; then
+    error_exit "Failed to configure Tails persistence."
+fi
+
+# Verify installation
+if ! command -v /usr/bin/securedrop-admin >/dev/null 2>&1; then
+    error_exit "Package installed but securedrop-admin command not found.\n\nInstallation may have failed."
+fi
+echo "- securedrop-admin command is available"
+
+NEW_CONFIG_DIR="$HOME/.config/securedrop-admin"
+
+# Copy site-specific config
+SITE_SPECIFIC_FILE="$OLD_CONFIG_DIR/group_vars/all/site-specific"
+if [[ -f "$SITE_SPECIFIC_FILE" ]]; then
+    cp "$SITE_SPECIFIC_FILE" "$NEW_CONFIG_DIR/"
+    echo "- Migrated: site-specific"
+
+    # Parse site-specific for GPG public key filenames and copy them
+    ossec_key=$(grep '^ossec_alert_gpg_public_key:' "$SITE_SPECIFIC_FILE" | awk '{print $2}' | tr -d "'\"")
+    if [[ -n "$ossec_key" && "$ossec_key" != "''" && -f "$OLD_CONFIG_DIR/$ossec_key" ]]; then
+        cp "$OLD_CONFIG_DIR/$ossec_key" "$NEW_CONFIG_DIR/"
+        echo "- Migrated: $ossec_key (OSSEC GPG public key)"
+    elif [[ -n "$ossec_key" && "$ossec_key" != "''" ]]; then
+        echo "! Not found (skipping): $ossec_key (OSSEC GPG public key)"
+    fi
+
+    securedrop_key=$(grep '^securedrop_app_gpg_public_key:' "$SITE_SPECIFIC_FILE" | awk '{print $2}' | tr -d "'\"")
+    if [[ -n "$securedrop_key" && "$securedrop_key" != "''" && -f "$OLD_CONFIG_DIR/$securedrop_key" ]]; then
+        cp "$OLD_CONFIG_DIR/$securedrop_key" "$NEW_CONFIG_DIR/"
+        echo "- Migrated: $securedrop_key (SecureDrop GPG public key)"
+    elif [[ -n "$securedrop_key" && "$securedrop_key" != "''" ]]; then
+        echo "! Not found (skipping): $securedrop_key (SecureDrop GPG public key)"
+    fi
+else
+    echo "! Not found (skipping): site-specific"
+fi
+
+# Copy Tor v3 keys
+if [[ -f "$OLD_CONFIG_DIR/tor_v3_keys.json" ]]; then
+    cp "$OLD_CONFIG_DIR/tor_v3_keys.json" "$NEW_CONFIG_DIR/"
+    echo "- Migrated: tor_v3_keys.json"
+else
+    echo "! Not found (skipping): tor_v3_keys.json"
+fi
+
+# Copy auth files
+for auth_file in app-journalist.auth_private app-ssh.auth_private mon-ssh.auth_private; do
+    if [[ -f "$OLD_CONFIG_DIR/$auth_file" ]]; then
+        cp "$OLD_CONFIG_DIR/$auth_file" "$NEW_CONFIG_DIR/"
+        echo "- Migrated: $auth_file"
+    else
+        echo "! Not found (skipping): $auth_file"
+    fi
+done
+
+# Copy source onion address
+if [[ -f "$OLD_CONFIG_DIR/app-sourcev3-ths" ]]; then
+    cp "$OLD_CONFIG_DIR/app-sourcev3-ths" "$NEW_CONFIG_DIR/"
+    echo "- Migrated: app-sourcev3-ths"
+else
+    echo "! Not found (skipping): app-sourcev3-ths"
+fi
+
```

Comment:
Instances with TLS certs will have 3 other files that need to be copied - the https crt, chain cert, and key files. If these values are set in `site-specific` they must be copied over.